src/jdk.sctp/unix/classes/sun/nio/ch/sctp/SctpChannelImpl.java

```diff
@@ -1098,8 +1081,8 @@ static native int send0(int fd, long address, int length,
     @SuppressWarnings("removal")
     private static void loadSctpLibrary() {
         IOUtil.load();   /* loads nio & net native libraries */
-        java.security.AccessController.doPrivileged(
-            new java.security.PrivilegedAction<Void>() {
+        AccessController.doPrivileged(
+            new java.security.PrivilegedAction<>() {
```

Comment:
I think we can import `java.security.PrivilegedAction` instead of using its full name